CHANGELOG.md

```diff
@@ -7,8 +7,9 @@
 * Update `RSpec/ExampleWording` cop to raise error for insufficient descriptions. ([@akrox58][])
 * Add new `RSpec/Capybara/NegationMatcher` cop. ([@ydah][])
 * Add `AllowedPatterns` configuration option to `RSpec/NoExpectationExample`.  ([@ydah][])
-* Improve `RSpec/NoExpectationExample` cop to ignore examples skipped or pending via metatada.  ([@pirj][])
+* Improve `RSpec/NoExpectationExample` cop to ignore examples skipped or pending via metadata.  ([@pirj][])
 * Add `RSpec/FactoryBot/ConsistentParenthesesStyle` cop. ([@Liberatys][])
+* Ignore example groups with `include_context` in `RSpec/LetBeforeExamples`. ([@pirj][])
```

Comment:
isn't it `include_examples` ? Also it ignores them only for auto-correct. So something along the lines `Do not attempt to auto-correct example groups with `include_examples` ) would be better. It would be good to link to the actual issue for context